include/rbdl/Body.h

```diff
@@ -111,19 +111,27 @@ struct RBDL_DLLAPI Body {
       return;
     }
 #else
-    if (other_body.mMass.is_zero() && other_body.mInertia == Math::Matrix3d::Zero()) {
+    if (other_body.mMass == 0. && other_body.mInertia == Math::Matrix3d::Zero()) {
       return;
     }
 #endif
 
     Math::Scalar other_mass = other_body.mMass;
     Math::Scalar new_mass = mMass + other_mass;
 
+#ifdef RBDL_USE_CASADI_MATH
```

Comment:
The macro #ifdef should only include the line 122 and not duplicate the whole block

Reply:
Also, this seems to be a mistake from before. The 
```
    return ;
}
```
block should be outside the #ifdef statement (which should only include the `if (...)` part. Can you change this please?